lib/foreman_rh_cloud/engine.rb

```diff
@@ -128,6 +128,14 @@ class Engine < ::Rails::Engine
         description: N_('Run remediation playbook generated by Insights'),
         host_action_button: false
       )
+
+      unless ForemanTasks::RecurringLogic.joins(:tasks).merge(
+        ForemanTasks::Task.where(label: 'InventorySync::Async::InventoryScheduledSync')
+      ).exists?
+        recurring_logic = ForemanTasks::RecurringLogic.new_from_cronline("0 0 * * *")
+        recurring_logic.save!
```

Comment:
This isn't really thread safe. Is that an issue here?

Reply:
I am not sure, but I suppose there shouldn't be many intializers running in parallel. If we see a problem with multiple schedules, we will need to solve it.